.github/workflows/maintenance.yml

```diff
@@ -8,6 +8,7 @@ on:
         type: choice
         options:
         - qa
+        - staging
```

Comment:
Probably another PR.
Should we also add `productiondata`?

Reply:
For maintenance page it's just relevant for production, and we add another env for testing (was qa). I'm adding staging as we are using that for the DR test.